include/forcing/Forcing.h

```diff
@@ -114,11 +122,14 @@ class Forcing
      */
     double get_next_hourly_precipitation_meters_per_second()
     {
-        //Increment forcing index
-        forcing_vector_index_ptr = forcing_vector_index_ptr + 1;
+        //Check forcing vector bounds before incrementing forcing index
+        if (forcing_vector_index_ptr < forcing_vector_size - 1)
+            //Increment forcing index
+            forcing_vector_index_ptr = forcing_vector_index_ptr + 1;
```

Comment:
This function side effects by incrementing `forcing_vector_index_ptr` which other get functions rely on, which is unclear.  Need to rethink this eventually...

Reply:
Ok, I will put a todo for this too.